chaindexing-postgres/src/drizzle/db.ts

```diff
@@ -0,0 +1,6 @@
+import { Pool } from 'pg';
+
+// TODO: Get connection string from env
+export const pool = new Pool({
+  connectionString: 'postgres://user:password@host:port/db'
```

Comment:
A bit early, but let's set the maximum parallel connections allowed:

```suggestion
  connectionString: 'postgres://user:password@host:port/db',
  max: 2
```